### DISCUSSION TITLE
Lab Assignment #5 👩‍💻

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/23S-ASIAAM-191A#20] [Assignments]
## Due 5/18 before class 

Please use this thread to post your [lab assignment's #4](https://albertkun.github.io/23S-ASIAAM-191A/assignments/week6/lab_assignment/) link, which is the GitHub pages URL. [Click here for a reminder on how to get it](https://albertkun.github.io/23S-ASIAAM-191A/labs/week1/3/#using-github-pages).

### For example

https://albertkun.github.io/22s-asiaam-191a-sample/week6/index.html

**Comment by paige-lee** [comment]
https://paige-lee.github.io/aa191a-labs/week6/index.html

**Comment by czhangy** [comment]
https://czhangy.github.io/ASIA-AM-191A/week6/maps.html

**Comment by zacharypakin** [comment]
https://zacharypakin.github.io/aa191a-23s/week06/maps.html

**Comment by joanneluong** [comment]
https://joanneluong.github.io/aas191a-labs/week6/index.html

**Comment by cassidytu** [comment]
https://cassidytu.github.io/asam191a-labs/week6/map.html

**Comment by SageLuong** [comment]
https://sageluong.github.io/23s-asianam-191a-labs/week6/animals.html

**Comment by shintuzd** [comment]
https://shintuzd.github.io/23s-asiaam-191a-assignments/week6/maps.html

**Comment by chrysanthemma** [comment]
https://chrysanthemma.github.io/asainam191a-labs/Lab4/index.html

**Comment by kmtsujimoto** [comment]
https://kmtsujimoto.github.io/23s-asiaam-191a-labs/week06/index.html

**Comment by vickydelk** [comment]
https://vickydelk.github.io/aa191-assignments/week6/index.html

**Comment by jamesfalkenroth** [comment]
https://jamesfalkenroth.github.io/23s-asianamerican-191a-labs/week6/index.html

**Comment by angelicaalacantar** [comment]
https://angelicaalacantar.github.io/aas191a-23s/week6/index.html

**Comment by danigold1020** [comment]
https://danigold1020.github.io/23s-asianam-191a-labs/week06/index.html

**Comment by ariscorpus** [comment]
https://ariscorpus.github.io/23s-asianam-191a-labs/week6/

**Comment by maizaha** [comment]
https://maizaha.github.io/23s-asiaam-191a-labs/week6/index.html

**Comment by cassiuscarandang** [comment]
https://cassiuscarandang.github.io/23s-asiaam-191a-lab1/week6/index.html

**Comment by qatle** [comment]
https://trinhqale.github.io/AM191A-labs/week6/index.html

**Comment by catherinehhu** [comment]
https://catherinehhu.github.io/191a-labs/week6/index.html

**Comment by enyi-emuka** [comment]
https://enyi-emuka.github.io/asiaam191a_labs/week7/index.html

**Comment by JonathanGLopez** [comment]
http://127.0.0.1:5500/week6/index.html

**Reply by JonathanGLopez** [reply]
Incorrect link. 
Correct link
https://jonathanglopez.github.io/Asiaam191aLB1/week6/index.html

**Comment by mandymwlai** [comment]
https://mandymwlai.github.io/AA-191-lab/week6/

**Comment by trufflelover** [comment]
https://trufflelover.github.io/23s-asiaam-191a-labs/lab5/

**Comment by stellakangj** [comment]
https://stellakangj.github.io/asiaam191a/week%206/index.html

**Comment by alanamalika** [comment]
https://alanamalika.github.io/23s-asianam-191a-labs/week06/index.html

**Comment by rebeccaksuen** [comment]
https://rebeccaksuen.github.io/23s-asiaam-191a-labs/week6/index.html
